test/algorithms/set_operations/sym_difference/sym_difference_areal_areal.cpp

```diff
@@ -0,0 +1,137 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2014-2015 Oracle and/or its affiliates.
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+// Contributed and/or modified by Menelaos Karavelas, on behalf of Oracle
+
+#include <iostream>
+
+#ifndef BOOST_TEST_MODULE
+#define BOOST_TEST_MODULE test_sym_difference_areal_areal
+#endif
```

Comment:
The sym-difference was already unit-tested by test_difference (which did both, a-b, b-a, and sym). Anyway, having this problem tackled individually is also OK